vnext/Microsoft.ReactNative/IReactDispatcher.idl

```diff
@@ -53,5 +53,20 @@ namespace Microsoft.ReactNative
       "Generally you can use @IReactContext.JSDispatcher to get the value of this property for "
       "a specific React instance.")
     static IReactPropertyName JSDispatcherProperty { get; };
+
+    DOC_STRING(
+      "Gets name of the `JSDispatcherTaskStartingEventName` property for the @IReactNotificationService.\n"
+      "This notification name is to be used with IReactNotificationService.")
+    static IReactPropertyName JSDispatcherTaskStartingEventName{ get; };
+
+    DOC_STRING(
+      "Gets name of the `JSDispatcherIdleWaitStartingEventName` property for the @IReactNotificationService.\n"
+      "This notification name is to be used with IReactNotificationService.")
+    static IReactPropertyName JSDispatcherIdleWaitStartingEventName{ get; };
+
+    DOC_STRING(
+      "Gets name of the `JSDispatcherIdleWaitCompletedEventName` property for the @IReactNotificationService.\n"
+      "This notification name is to be used with IReactNotificationService.")
```

Comment:
@xidachenfb Looks like this is missing the definite article "the" between Gets and name ("Gets the name")

@stevewhims fyi in case you have any feedback

Reply:
Thanks for the comments. I will have another PR to fix that.

Reply:
thanks!